control-plane-operator/controllers/hostedcontrolplane/autoscaler/reconcile.go

```diff
@@ -167,9 +167,7 @@ func ReconcileAutoscalerDeployment(deployment *appsv1.Deployment, hcp *hyperv1.H
 							},
 							InitialDelaySeconds: 60,
 							PeriodSeconds:       60,
-							SuccessThreshold:    1,
-							FailureThreshold:    5,
-							TimeoutSeconds:      5,
+							TimeoutSeconds:      30,
```

Comment:
Can you walk through your thought process for these changes? In this stanza, for instance, the previous FailureThreshold was higher than the default you set, why did it get removed? Similarly, if unset the default timeout is 30s, so why specify it explicitly?

Reply:
Aligning with the defaults in probes.go.  `autoscaler` and `ignitionserver` do not use `DeploymentConfig` to set up the probes and, instead, declare them inline in the deployment spec.  We should really just to covert these two to use `DeploymentConfig` to set up the probes.